frontend/src/metabase/sharing/components/AddEditSidebar/CaveatMessage.jsx

```diff
@@ -1,25 +1,12 @@
 import React from "react";
 import { t } from "ttag";
 
-import MetabaseSettings from "metabase/lib/settings";
-
 import Text from "metabase/components/type/Text";
-import ExternalLink from "metabase/core/components/ExternalLink";
 
 function CaveatMessage() {
   return (
     <Text className="mx4 my2 p2 bg-light text-dark rounded">
-      <span className="text-bold">{t`Note`}:&nbsp;</span>
-      {t`charts in your subscription won't look the same as in your dashboard.`}
-      &nbsp;
-      <ExternalLink
-        className="link"
-        target="_blank"
-        href={MetabaseSettings.docsUrl("dashboards/subscriptions")}
-      >
-        {t`Learn more`}
-      </ExternalLink>
-      .
+      {t`Charts delivered via your subscription may look slightly different from the charts in your dashboard, but the data will be the same.`}
```

Comment:
I'd suggest we omit the bit about the data being the same, since in paid plans you can add subscription-specific filters. Also edited a bit for brevity.

```suggestion
      {t`Charts in subscriptions may look slightly different from charts in dashboards.`}
```